### ISSUE TITLE
Houston crashing when using IronRouter

### ISSUE THREAD

**Issue by jeriscc** [gterrono/houston#322]
After installing Houston for the simple-todo tutorial app, a message appear in the application:
" Oops, looks like there's no route on the client or the server for url: "http://localhost:3000/." "

To solve this, I installed IronRouter and defined the "/" route as home. After that, when I went to houston admin again, the admin system does work anymore.

**Comment by Menda** [comment]
Yes, I also get this error using FlowRouter as my main router. I get this "Oops" error in every address outside the `/admin`. Any solution?
